src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/multiclass_nms/multiclass_nms_kernel_ref.cpp

```diff
@@ -0,0 +1,162 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "multiclass_nms_kernel_ref.h"
+
+#include <algorithm>
+#include <string>
+
+#include "kernel_selector_utils.h"
+
+namespace kernel_selector {
+
+ParamsKey MulticlassNmsKernelRef::GetSupportedKey() const {
+    ParamsKey k;
+
+    k.EnableInputDataType(Datatype::F16);
+    k.EnableInputDataType(Datatype::F32);
+    k.EnableInputDataType(Datatype::INT32);
+    k.EnableInputDataType(Datatype::INT64);
+    k.EnableOutputDataType(Datatype::F16);
+    k.EnableOutputDataType(Datatype::F32);
+    k.EnableOutputDataType(Datatype::INT32);
+    k.EnableOutputDataType(Datatype::INT64);
```

Comment:
do you need int for output? as I understand indices is "mutable" input?

Reply:
Done.